### DISCUSSION TITLE
Would you like to be assigned Issues to work on? (PLEASE READ THE BODY)

### DISCUSSION THREAD

**Discussion by austinsonger** [Secure-Compliance-Solutions-LLC/GVM-Docker#193] [General]
We would like to continue to improve this repo at a faster pace and to finish some of our [milestones](https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/milestones) I've set for this project. 

@pixelsquared and myself have a lot of work for our daytime job (and tons of other repos that we are working on, it would be nice to pull some of the frequent users (and users who have submitted pull requests in the past) of the repo into the development and the improvement of this GVM docker container as well as our other repo [OpenVAS-Docker](https://github.com/Secure-Compliance-Solutions-LLC/OpenVAS-Docker) that relates to this repo.

### Would you like to be assign issues to work on?

**Template for Answering**
```
**Name** *(First Name if you are uncomfortable giving full name)* : `Name`
**Username**: `Username`
**Country of Origin**: `Country`
**State** *(If in United States)*:  `State`

ANSWER: <YES or NO>

If Yes, Please Rank Yourself for Experience From the following (`1` being **Novice** experience to `10` Being **Professional**)


**Dockerfile**:`#`
**Docker Compose**:`#`
**Bash**:`#`
**Greenbone Vulnerability Manager**:`#`
**OpenVAS**:`#`

On a weekly basis, how many Issues would you be willing to work on:

`#` issues a week.

If your job consist of any of the following work duties or tasks
- Docker
- Kubernetes
- Programming
- Cyber Security
- System Administration
- DevOps

Please add your job title:

**Job Title**:`         `
**Years in Current Role**:`#`

```

----------------------
> **_THIS IS WHAT IT SHOULD LOOK LIKE WHEN REPLYING TO THIS_**.

**Name** *(First Name if you are uncomfortable giving full name)*: `John Doe`
**Username**: `JohnNoneYaBusinessDoe`
**Country of Origin**: `United States`
**State** *(If in United States)*: `Washington State`

ANSWER: YES or NO

If Yes, Please Rank Yourself for Experience From the following (`1` being **Novice** experience to `10` Being **Professional** experience)

**Dockerfile**:`10`
**Docker Compose**:`6`
**Bash**:`7`
**Greenbone Vulnerability Manager**:`6`
**OpenVAS**:`9`

On a weekly basis, how many Issues would you be willing to work on:

`2` issues a week.

If your job consist of any of the following work duties or tasks
- Docker
- Kubernetes
- Programming
- Cyber Security
- System Administrator

Please add your job title:

**Job Title**:`Security Operations Engineer`
**Years in Current Role**:`10`

**Comment by mrod23** [comment]
**Name** ** : michael
**Username**: mrod23
**Country of Origin**: USA
**State** *(If in United States)*:  NY

ANSWER: <YES or NO>

If Yes, Please Rank Yourself for Experience From the following (`1` being **Novice** experience to `10` Being **Professional**)


**Dockerfile**:8
**Docker Compose**:8
**Bash**:8
**Greenbone Vulnerability Manager**:5
**OpenVAS**:5

On a weekly basis, how many Issues would you be willing to work on:

`1` issues a week.

If your job consist of any of the following work duties or tasks

All 4 plus years 

- Docker - yes
- Kubernetes - yes
- Programming - yes
- Cyber Security - yes
- System Administration - yes
- DevOps - yes

Please add your job title:

**Job Title**:`   Senior Software Engineer
**Years in Current Role**:`4`

**Comment by Dexus** [comment]
**Name** *(First Name if you are uncomfortable giving full name)* : `Josef`
**Username**: `Dexus`
**Country of Origin**: `DE`

ANSWER: 

If Yes, Please Rank Yourself for Experience From the following (`1` being **Novice** experience to `10` Being **Professional**)


**Dockerfile**:`9`
**Docker Compose**:`9`
**Bash**:`8`
**Greenbone Vulnerability Manager**:`5`
**OpenVAS**:`5`

On a weekly basis, how many Issues would you be willing to work on:

`1-2` issues a week.

If your job consist of any of the following work duties or tasks

- Docker
- Kubernetes
- Programming
- Cyber Security
- System Administration
- DevOps

all of them 5+ yrs

Please add your job title:

**Job Title**:`Linux Consultant`
**Years in Current Role**:`5+`
